frontend/public/components/utils/__tests__/firehose.data.tsx

```diff
@@ -1,6 +1,6 @@
 import { Map as ImmutableMap } from 'immutable';
 
-export { PodModel } from '../../../models';
+export { PodModel, ServiceModel } from '../../../models';
```

Comment:
what is this changed targeted for?

Reply:
> what is this changed targeted for?

Oh, that wasn't needed. Removed it. Thanks for the catch!